tests/unit/render/test_vega_converter.py

```diff
@@ -248,3 +248,17 @@ def test_convert(
 
     assert datapoints == expected_datapoints
     assert resolved_properties == expected_properties
+
+
+@pytest.mark.parametrize(
+    "properties,label",
+    [
+        ({"x": {"actual.csv": "actual"}}, "actual"),
+        (
+            {"x": {"train_actual.csv": "actual", "val_actual.csv": "actual"}},
+            "actual",
+        ),
+    ],
+)
+def test_infer_x_label(properties, label):
```

Comment:
@dberenbaum I added this test to try to understand `infer_x_label` as I am not really sure about the expected behavior for all the cases.

Could you add an example case here for the change introduced in this P.R.?

Reply:
Thanks for the skeleton! Added a test for the default x label.